SofaScoreAcademy/SofaScoreAcademy/MockData.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  MockData.swift
+//  SofaScoreAcademy
+//
+//  Created by Dominik Prce on 08.04.2024..
+//
+
+import Foundation
+
+struct MockData {
+    
+    static func exampleEvents(forSport sportSlug: String) -> [Event] {
+        switch sportSlug {
+        case "football":
+            return [
+                Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", teamOneScore: 1, teamTwoScore: 2, startTime: Constants.startTime1, matchStatus: .isFinished),
+                Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", teamOneScore: 0, teamTwoScore: 1, startTime: Constants.startTime2, matchTime: 36, matchStatus: .inProgress),
+                Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", startTime: Constants.startTime3, matchStatus: .notStarted),
+                Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", startTime: Constants.startTime4, matchStatus: .notStarted)
+            ]
+        case "basketball":
+            return [
+                Event(teamOneName: "New Orleans Pelicans", teamTwoName: "Phoenix Suns", teamOneScore: 29, teamTwoScore: 34, startTime: Constants.startTime1, matchStatus: .isFinished),
+                Event(teamOneName: "New Orleans Pelicans", teamTwoName: "Phoenix Suns", teamOneScore: 6, teamTwoScore: 10, startTime: Constants.startTime2, matchTime: 36, matchStatus: .inProgress),
+                Event(teamOneName: "New Orleans Pelicans", teamTwoName: "Phoenix Suns", startTime: Constants.startTime3, matchStatus: .notStarted),
+                Event(teamOneName: "New Orleans Pelicans", teamTwoName: "Phoenix Suns", startTime: Constants.startTime4, matchStatus: .notStarted)
+            ]
+        case "american-football":
+            return [
+                Event(teamOneName: "Alabama State Hornets", teamTwoName: "Bryant Bulldogs", teamOneScore: 30, teamTwoScore: 38, startTime: Constants.startTime1, matchStatus: .isFinished),
+                Event(teamOneName: "Alabama State Hornets", teamTwoName: "Bryant Bulldogs", teamOneScore: 14, teamTwoScore: 19, startTime: Constants.startTime2, matchTime: 36, matchStatus: .inProgress),
+                Event(teamOneName: "Alabama State Hornets", teamTwoName: "Bryant Bulldogs", startTime: Constants.startTime3, matchStatus: .notStarted),
+                Event(teamOneName: "Alabama State Hornets", teamTwoName: "Bryant Bulldogs", startTime: Constants.startTime4, matchStatus: .notStarted)
+            ]
+        default:
+            return []
+        }
+    }
+    
+    static func exampleLeague(forSport sportSlug: String) -> League? {
```

Comment:
kaj si spominjao da nemamo slike za lige? ako imamo dodaj da se i to mijenja tu

Reply:
+